Exercises/JS-30/10-allowOneFunctionCall/main.ts

```
@@ -0,0 +1,33 @@
+
+type JSONValue =
+  | null
+  | boolean
+  | number
+  | string
+  | JSONValue[]
+  | { [key: string]: JSONValue };
+type OnceFn = (...args: JSONValue[]) => JSONValue | undefined;
+
+// @ts-ignore Intentional broad fn() definition.)
+export function once(fn: Function): OnceFn {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)